platform-hub-api/db/migrate/20171121125850_add_kubecfg_extra_fields_to_kubernetes_clusters.rb

```diff
@@ -0,0 +1,6 @@
+class AddKubecfgExtraFieldsToKubernetesClusters < ActiveRecord::Migration[5.0]
+  def change
+    add_column :kubernetes_clusters, :api_url, :string, null: false
+    add_column :kubernetes_clusters, :ca_cert_encoded, :string, null: false
```

Comment:
I _think_ this migration will fail for existing data, as existing clusters won't have these fields set, and the they are required to be non-null.

Reply:
Yeah, I was thinking to make them optional and then we don't show the generate kubecfg button if the values are null. What do you think? Or should we somehow fill existing records with dummy data / set empty strings?